cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py

```diff
@@ -0,0 +1,2773 @@
+    """
+    Add SCVMM-2016 provider.
+    Perform SSA on RHEL 7.4 VM.
+    Check whether Packages retrieved.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_packages_scvmm2k16_windows2012r2_ntfs():
+    """
+    Add SCVMM-2016 provider.
+    Perform SSA on Windows 2012 server R2 VM having NTFS filesystem.
+    Check whether it retrieves Applications.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_packages_ec2_windows2012r2_ntfs():
+    """
+    Add EC-2 provider.
+    Perform SSA on Windows 2012 server R2 VM having NTFS filesystem.
+    Check whether it retrieves Applications.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.9
+        tags: smartstate
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_packages_scvmm2k16_windows2016_ntfs():
+    """
+    Add SCVMM-2016 provider.
+    Perform SSA on Windows 2016 server VM having NTFS filesystem.
+    Check whether it retrieves Applications.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_packages_azure_rhel():
+    """
+    1. Add Azure provider
+    2. Perform SSA on RHEL instance.
+    3. Check Packages are retrieved.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/3h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_packages_azure_windows2012r2_ntfs():
+    """
+    1. Add Azure provider
+    2. Perform SSA on Windows server 2012 R2.
+    3. Check Packages are retrieved.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/3h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_packages_scvmm2k12_centos_xfs():
+    """
+    Add SCVMM-2012 provider.
+    Perform SSA on CentOS VM.
+    Check whether Packages retrieved.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(2)
```

Comment:
You are using tier(2) only in this case, every other test case uses tier(1), so you can add tier mark to global pytestmark.

Reply:
I have added global tier for it.